sitewit/services.py

```diff
@@ -25,10 +25,17 @@ def __init__(self, **kwargs):
 
         super(SitewitService, self).__init__(**kwargs)
 
-    def _get_auth_header(self, account_token=None):
+    def _get_account_auth_header(self, account_token):
+        auth_raw = '%s:%s:%s' % (
+            self._partner_id, self._partner_token, account_token)
```

Comment:
Yes, this is a possible option. Do you think it is better, or just another way to do the same?